lib/v2/components/amount_entry/amount_entry_widget.dart

```diff
@@ -32,7 +32,7 @@ class AmountEntryWidget extends StatelessWidget {
         listenWhen: (_, current) => current.pageCommand != null,
         listener: (context, state) {
           // ignore: cast_nullable_to_non_nullable
-          onValueChange((state.pageCommand as SendTextInputDataBack).textToSend);
+          onValueChange((state.pageCommand as SendTextInputDataBack).textToSend.replaceAll(',', ''));
```

Comment:
this should be in the state mapper or w.e you create this SendTextInputDataBack.

Reply:
you are right, fix